config.php

```diff
@@ -38,6 +38,25 @@
 		'32'	=> 616,
 	),
 
+	/**
+	* IDs of forums used for extension/styles validation queue topics
+	*/
+	'titania_extensions_queue' => array(
+		ext::TITANIA_QUEUE_DISCUSSION => 516,
+		ext::TITANIA_QUEUE            => 511,
+		'trash'                       => 521,
```

Comment:
Maybe we need a constant for this too? Seems strange to have constants for everything but this.

Reply:
All the constants in this extension seem to be just for magic numbers.